assets/src/apps/app.tsx

```diff
@@ -7,6 +7,19 @@ import { configureStore } from 'state/store';
 import { Maybe } from 'tsmonad';
 import { b64DecodeUnicode } from 'utils/decode';
 
+export function defineEmbeddableReactComponent<T extends State>(
```

Comment:
I'd probably prefer to refactor the ActivityBankApp instead of a wrapper like this since it's not really needed anymore.